LICENSE-MIT

```diff
@@ -1,7 +1,3 @@
-MIT License
-
-Copyright (c) 2022-2022 Rust-marker
```

Comment:
This feels a bit weird, especially since the [official license text](https://opensource.org/licenses/MIT) includes the Copyright line. Though I understand that "Rust Developers" as in the linked commits is not a legal entity and would in theory mean that every dev has a copy right. :thinking: I'd think that Rust-Marker would make the Rust-Marker organization the copy right holder which kind of makes sense. But looking at other licenses under rust-lang it seems like they also removed it, so I would be fine with this